winui/Kanban/swimlane.md

```diff
@@ -0,0 +1,359 @@
+---
+layout: post
+title: Swimlane support in WinUI Kanban control | Syncfusion
+description: Learn here all about the swimlane support and its customization in Syncfusion WinUI Kanban (SfKanban) control.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+
+# Swimlane in WinUI Kanban (SfKanban) control
+
+Swimlanes are horizontal categorizations of cards on the WinUI Kanban Board. It is used for grouping of cards, which brings transparency to the workflow process They allow you to categorize your workflow by projects, teams, users, or any other criteria you need.
+
+By default, swimlanes are categorized based on the `Assignee` values in the `KanbanModel` class. You can also define a category by mapping `SwimlaneKey` to the appropriate property name in the defined `KanbanModel`.
+
+The following code example demonstrates how to group the underlying data collection using the `SwimlaneKey` property.
+
+{% tabs %}
+{% highlight XAML hl_lines="3" %}
+
+<kanban:SfKanban x:Name="kanban"
+                    ItemsSource="{Binding TaskDetails}"
+                    SwimlaneKey="IndicatorColorKey">
```

Comment:
SwimlaneKey = ""
ItemsSource =

Follow this order and change highlight value basedon changes

Reply:
Changed.